src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -357,11 +359,21 @@ private void testUpload(byte[] content, boolean encrypted, Map<String,String> us
         verifyUpload(ex3ConfigForStudy, uploadId, filename, true, expectedMetadata);
 
         // Verify the record in Bridge.
-        ForWorkersApi workersApi = adminDeveloperWorker.getClient(ForWorkersApi.class);
-        HealthDataRecordEx3 record = workersApi.getRecordEx3(IntegTestUtils.TEST_APP_ID, uploadId).execute().body();
+        ForConsentedUsersApi usersApi = user.getClient(ForConsentedUsersApi.class);
+        HealthDataRecordEx3 record = usersApi.getRecordEx3ById(uploadId, "true").execute().body();
         assertTrue(record.isExported());
         DateTime oneHourAgo = DateTime.now().minusHours(1);
         assertTrue(record.getExportedOn().isAfter(oneHourAgo));
+
+        // Verify the presigned download url for a health record is generated and contains the data expected.
+        String url = record.getDownloadUrl();
+        HttpResponse responseForPresignedUrl = Request.Get(url).execute().returnResponse();
+        String contentForPresignedUrl = EntityUtils.toString(responseForPresignedUrl.getEntity());
+        assertEquals(200, responseForPresignedUrl.getStatusLine().getStatusCode());
+        assertEquals(UPLOAD_CONTENT, contentForPresignedUrl.getBytes(StandardCharsets.UTF_8));
+
+        DateTime fiftyNineMinsAfter = DateTime.now().plusMinutes(59);
```

Comment:
The clock skew on Jenkins can sometimes go up to 10 minutes. We should change this to plusMinutes(50) just to be safe.